### DISCUSSION TITLE
managed identities support

### DISCUSSION THREAD

**Discussion by mo-saeed** [Azure/azure-workload-identity#472] [General]
Hello,

now we are using aad-pod-identity with managed identities. so we create a managed identity for each namespace (connected to resource group) and it will be a pain for us to start now using AAD applications instead. 

Do you have any estimation when this will be supported. and till this happens should aad-pod-identity be supported ?

Thanks

**Comment by aramase** [comment]
The feature is being tracked here: https://github.com/Azure/azure-workload-identity/issues/325. The support is required from AAD and it is currently in private preview. There are estimates for timeline in the issue.
